src/Components/Banner/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { View, Text, ImageBackground } from 'react-native';
+
+import { Image } from '../../Components';
+import { Pill } from '../../Components';
+import { IProductonOffer } from './types';
+import styles from './styles';
+
+const Banner: React.FC<IProductonOffer> = ({
+  name,
+  endDate,
+  price,
+  newPrice,
+  imageLink,
+}) => {
+  return (
+    <View style={styles.container}>
+      <View>
+        <Text style={[styles.text, styles.title]}>{name}</Text>
+
+        <Text style={[styles.text, styles.endDate]}>Válido até {endDate}</Text>
+
+        <View style={styles.oldPriceView}>
+          <Text style={[styles.text, styles.textPrice]}>De </Text>
+          <Text style={[styles.text, styles.textPrice, styles.oldPrice]}>
+            {price}
+          </Text>
+        </View>
+
+        <Pill title={`Por R$ ${newPrice}`} />
```

Comment:
Outro ponto importante aqui é que é interessante tratar o valor do `newPrice` e do `price` para colocar os centavos no produto, para caso tenhamos produtos com valores pequenos e os valores dos centavos seja importante para o valor total da promoção.
Nessa linha bastaria você encapsular o `newPrice` com um método para tratar o valor:
```suggestion
        <Pill title={`Por R$ ${handleMoneyFormat(newPrice)}`} />
```
E nessa função `handleMonetaryFormat` tratar devidamente o valor recebido, considerando os cenários de:
- Receber um valor sem casas decimais:
`
<Banner
     name="test"
     price={30}
     imageLink="https://facebook.github.io/watchman/static/logo.png"
     newPrice={25}
     endDate="25/08"
/>
`
Sendo necessário convertê-lo para string e adicionar um `,00` no fim do valor;
- Receber um valor number com casas decimais:
`
<Banner
     name="test"
     price={30}
     imageLink="https://facebook.github.io/watchman/static/logo.png"
     newPrice={25}
     endDate="25/08"
/>
`
Sendo necessário convertê-lo para string e substituir o separador `.` por `,`, como padrão BR.